src/OpenTelemetry.Shared/PersistentStorageHelper.cs

```diff
@@ -0,0 +1,223 @@
+// <copyright file="PersistentStorageHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace OpenTelemetry.Shared
+{
+    internal static class PersistentStorageHelper
```

Comment:
Should we create a helper folder to facilitate in the future?

Reply:
Got only one file, if we get a second helper class we could consider moving to different folder.